controllers/cinderapi_controller.go

```diff
@@ -856,7 +891,14 @@ func (r *CinderAPIReconciler) reconcileNormal(ctx context.Context, instance *cin
 			instance.Status.ReadyCount,
 		)
 		if err != nil {
-			return ctrl.Result{}, err
+			error := fmt.Errorf("verifying API NetworkAttachments (%s) %w", instance.Spec.NetworkAttachments, err)
```

Comment:
Isn't this handled already at L911 below?

Reply:
oh, right, my bad! The logic is so squirrely (too many conditionals) and potential exit points.